blas/testblas/common_test.go

```diff
@@ -2,8 +2,10 @@ package testblas
 
 import (
 	"math"
+	"math/cmplx"
 	"testing"
 
+	"golang.org/x/exp/rand"
```

Comment:
This should be in its own import stanza between std and us.

Reply:
Done.